Tools/RemoteConsole/ly_remote_console/ly_remote_console/remote_console_commands.py

```diff
@@ -162,7 +162,8 @@ def _connect_to_port(self, timeout_port, retry_delay):
                     # failed to find RC on a port, wait
                     time_waited += retry_delay
                     time.sleep(retry_delay)
-                    logger.debug("remote_console_commands.py:start: Have waited for at least {} seconds.".format(time_waited))
+                    logger.debug(f'remote_console_commands.py:start: Have waited for at least {time_waited} seconds.')
+                    print(f'remote_console_commands.py:start: Have waited for at least {time_waited} seconds.')
```

Comment:
Remove duplicate print

Reply:
I added it intentionally, because the logger logs to a file, nothing on screen.

Reply:
Then we need to find a different solution here.  Duplicate prints are a pain to maintain, and add noise.  Either remove them or let's figure a way to get the logger to print to screen as well.

Reply:
We do not want the logger prints on screen, because there is a client message log feed, it comes too fast.
To make it nice I can use two loggers, one to file, one to console. They can be configured by the caller.

Reply:
How do other python scripts/applications do this?  I'm pretty sure they _don't_ maintain two separate loggers ;)
Please research & implement that.  Again, we do not want to maintain two separate loggers.  No-one wants to, nor will remember, to add two prints in two separate systems whenever they add new functionality.